plugins/MantisGraph/MantisGraph.php

```diff
@@ -113,6 +113,7 @@ function resources() {
 			html_javascript_link( 'chart-' . CHARTJS_VERSION . '.min.js' );
 			html_javascript_link( 'chart.bundle-' . CHARTJS_VERSION . '.min.js' );
 		}
+		html_javascript_link( '../plugins/MantisGraph/MantisGraph.js' );
```

Comment:
> We could use chart-xxxx.min.js for some dashboard functionality (issue trend, ...) on "My View" page. So maybe better not to move.

That was my question about. I don't know if they are used system-wide.
In that case, the inclusion of those js, should be done by core, not by the plugin?